test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,49 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import static org.junit.Assert.*;
+
+public class SortCommandTest {
+
+    private AddressBook addressbook;
+    private List<Person> expectedOrderList = new ArrayList<Person>();
+
+    @Test
+    public void execute_sortCommand() {
+        SortCommand command = new SortCommand();
+        Person a = TestUtil.generateTestPerson("a");
+        Person b = TestUtil.generateTestPerson("b");
+        Person c = TestUtil.generateTestPerson("c");
+        addressbook = TestUtil.createAddressBook(c, b, a);
```

Comment:
While this works, let's try to populate our test data with more realistic names instead!